src/utils/database.js

```diff
@@ -0,0 +1,65 @@
+const DB = {
+  USERS: [],
+  BOARDS: [],
+  TASKS: [],
+};
+
+class TableSelector extends Array {
+  select(fieldName, fieldValue) {
+    const result = this.filter(item => item?.[fieldName] === fieldValue) || [];
```

Comment:
why you use `|| []` ?

Reply:
@kovalenkoiryna15 You're right, it's unnecessary. Just missed it after refactoring :)